frontend/projects/ui/src/app/pages/apps-routes/app-show/app-show.page.html

```diff
@@ -52,21 +52,17 @@
           <ion-row class="ion-align-items-center">
             <ion-col class="ion-text-center">
               <h2>
-                <ion-text color="warning">
-                  You are using an unencrypted http connection
-                </ion-text>
+                <ion-text color="warning">Http detected</ion-text>
               </h2>
               <p class="ion-padding-bottom">
-                Click the button below to switch to https. Your browser may warn
-                you that the page is insecure. You can safely bypass this
-                warning. It will go away after you
+                Your connection is insecure.
                 <a
-                  [routerLink]="['/system', 'lan']"
+                  href="https://docs.start9.com/latest/user-manual/connecting/connecting-lan/"
```

Comment:
We ideally should update these links with the proper page in the new docs refactor before release, or at least need to make sure that page will no longer exist so a redirect can be implemented.

Reply:
also we dont additionally want to link to where to download the cert here?

Reply:
I believe the docs are changing quite a bit, so redirects will be necessary everywhere. If you know the new link and are confident it will be done in time for this release, you can change it here.

I'll take another look, but givent he choice I would rather link to the instructions than to the cert page. But that reminds me, we should rename the LAN page

Reply:
Reverted to local links